### PR TITLE
tests: move main/ubuntu-core-* tests to core/ suite

### PR DESCRIPTION
**Pull request by anonymouse64**
This is a follow-up to https://github.com/snapcore/snapd/pull/8106, and note that this does not enable any new systems for these tests, it just moves them over.

We will work on enabling these tests to run on other systems in a follow-up, I just think it will be easier to move everything first, then enable piecemeal in individual PR's as necessary.

### CODE REVIEW

**Comment by pedronis** [comment]
note that there are tests not called ubuntu-core-*  that are also core only, we likely do want to move most of those as well

```
core-watchdog/task.yaml:systems: [ubuntu-core-16-*]
create-user/task.yaml:systems: [ubuntu-core-1*]
enable-disable-units-gpio/task.yaml:systems: [ubuntu-core-1*-64]
failover/task.yaml:systems: [ubuntu-core-16-*]
gadget-update-pc/task.yaml:systems: [ubuntu-core-1*]
remodel-base/task.yaml:systems: [ubuntu-core-18-64]
remodel-gadget/task.yaml:systems: [ubuntu-core-18-64]
remodel-kernel/task.yaml:systems: [ubuntu-core-16-64]
remodel/task.yaml:systems: [ubuntu-core-1*-64]
snap-auto-import-asserts-spools/task.yaml:systems: [ubuntu-core-1*-64]
snap-auto-import-asserts/task.yaml:systems: [ubuntu-core-1*-64]
snap-auto-mount/task.yaml:systems: [ubuntu-core-1*-64]
snap-core-fixup/task.yaml:systems: [ubuntu-core-1*]
snap-core-symlinks/task.yaml:systems: [ubuntu-core-1*]
snap-set-core-config/task.yaml:systems: [ubuntu-core-1*]
```

**Comment by anonymouse64** [comment]
@pedronis I added those tests and some more I found which are UC specific. There are some cleanups we should do too, like create-user and create-user-2 here seem to be testing essentially the same thing?

Also since this was such a small PR, I just force-pushed it to pick up the master fixes from this AM

File: tests/core/core-watchdog/task.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ summary: Check that the core.watchdog settings work
 environment:
```
Comment (pedronis) [inline tests/core/core-watchdog/task.yaml]: should this test be renamed just watchdog ?

File: tests/core/snap-core-symlinks/task.yaml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 summary: Check that the seed symlinks work
```
Comment (pedronis) [inline tests/core/snap-core-symlinks/task.yaml]: maybe rename this test to seed-base-symlinks ?

**Pull request merged**